_attributes/common-attributes.adoc

```diff
@@ -335,3 +335,5 @@ endif::openshift-origin[]
 // Hosted control planes related attributes
 :hcp-capital: Hosted control planes
 :hcp: hosted control planes
+:mce: multicluster engine for Kubernetes operator
+:mce-short: multicluster engine operator
```

Comment:
The supplementarty style guide requires capitalizing "Operator": https://redhat-documentation.github.io/supplementary-style-guide/#operator . This requirement appears to apply to both attributes here.